### PR TITLE
update devnet tutorial

### PR DESCRIPTION
**Pull request by felipemadero**

### CODE REVIEW

File: examples/tokenvm/DEVNETS.md [hunk 1]
```diff
@@ -12,17 +12,17 @@ page](https://github.com/ava-labs/avalanche-ops/releases/tag/latest).
 #### Option 1: Install `avalanche-ops` on Mac
 ```bash
 rm -f /tmp/avalancheup-aws
-wget "https://github.com/ava-labs/avalanche-ops/releases/download/latest/avalancheup-aws.aarch64-apple-darwin"
```
Comment (patrick-ogrady) [inline examples/tokenvm/DEVNETS.md:15]: I prefer to use aarch64 (for new M-series)

Comment (patrick-ogrady) [reply examples/tokenvm/DEVNETS.md:15]: That being said, I think M-series can still run x86_64 using emulation.

File: examples/tokenvm/DEVNETS.md [hunk 2]
```diff
@@ -80,12 +81,12 @@ Now we can spin up a new network of 6 nodes with some defaults:
...
 /tmp/avalancheup-aws default-spec \
 --arch-type amd64 \
---rust-os-type ubuntu20.04 \
+--os-type ubuntu20.04 \
 --anchor-nodes 3 \
 --non-anchor-nodes 3 \
 --regions us-west-2 \
 --instance-mode=on-demand \
---instance-types=c5.4xlarge \
+--instance-types='{"us-west-2":["c5.4xlarge"]}' \
```
Comment (patrick-ogrady) [inline examples/tokenvm/DEVNETS.md:89]: A lot of these changed in other branches, thanks for reminding me this was broken.

File: examples/tokenvm/DEVNETS.md (hunk 1 above)
Comment (felipemadero) [reply examples/tokenvm/DEVNETS.md:15]: ok I think that's correct emulators can be used, anyway will go back on this. I do also prefer to use aarch64 :D

File: examples/tokenvm/DEVNETS.md [hunk 3]
```diff
@@ -37,6 +37,7 @@ use while deploying your devnet.
 ### Step 3: Install `token-cli`
 ```bash
 export ARCH_TYPE=$(uname -m)
+[ $ARCH_TYPE = x86_64 ] && ARCH_TYPE=amd64
```
Comment (patrick-ogrady) [inline examples/tokenvm/DEVNETS.md:40]: What does this do? (I haven't needed it)

**Review by patrick-ogrady — APPROVED** [review]

**Pull request merged**